data/swtpm/ssh_script_2

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/expect
+ 
+#Usage sshsudologin.expect <host> <ssh user> <ssh password> <su user> <su password>
+
+set timeout 60
+spawn ssh [lindex $argv 1]@[lindex $argv 0]
+expect "yes/no*" {
+    send "yes\r"
+    expect "*?assword" { send "[lindex $argv 2]\r" }
+    } "*?assword" { send "[lindex $argv 2]\r" }
+
+expect " #" { send "ls /dev/tpm*\r" }
+expect " #" { send "tpm2_pcrread sha1:0\r" }
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details.

It seems like the only difference between above two scripts is `tpm_version` vs. `tpm2_pcrread sha1:0`. You can generalize one script and use an environment variable, e.g. `$env(MY_VARIABLE)` and then call like:


```bash
MY_VARIABLE="tpm_version" data/swtpm/ssh_script
MY_VARIABLE="tpm2_pcrread sha1:0" data/swtpm/ssh_script

Reply:
Agree with you, let me enhance it

Reply:
Modified